core/vm/evm.go

```diff
@@ -425,7 +428,8 @@ func (evm *EVM) StaticCall(caller ContractRef, addr common.Address, input []byte
 	// This doesn't matter on Mainnet, where all empties are gone at the time of Byzantium,
 	// but is the correct thing to do and matters on other networks, in tests, and potential
 	// future scenarios
-	evm.StateDB.AddBalance(addr, big0)
+	// This fine against overflow, since amount is zero
+	_ = evm.StateDB.AddBalance(addr, big0)
```

Comment:
`AddBalance` should return a nil error here, would it be better to check the error value and rely on the fact that it returns nil (with the value 0) rather than ignore the error in the caller?